.github/ISSUE_TEMPLATE/user-story-template.md

```diff
@@ -24,6 +24,7 @@ Given some context, when (X) action occurs, then (Y) outcome is achieved
 - [ ] Can misuse of this feature cause harm? If yes, create an alert
 - [ ] Update the status of related findings, insights, and hypotheses on the Research Airtable
 - [ ] Once change/fix/feature is implemented, link relevant Airtable records to design artifacts (Figma)
+- [ ] Does the feature change our IA? If so, update the Sitemap in En/Fr
```

Comment:
What is IA?

Reply:
Information Architecture. I'll spell it out, good catch!